single_cell_RNAseq/pipeline_pre_qc.nf

```diff
@@ -187,7 +194,8 @@ workflow {
                 FREEMUXLET_LIBRARY(ch_single_lib_transformed)
 
                 ch_sample_map = FREEMUXLET_LIBRARY.out.sample_map
-            }
+                ch_lib_vcf =  FREEMUXLET_LIBRARY.out.vcf
```

Comment:
so I think you can take this line and lines 180-186 and move them to the conditional below.